harness_tests/system-diagnostic/main1.c

```diff
@@ -0,0 +1,61 @@
+#include <stdlib.h>
+#include <test/test.h>
+#include <uptime/uptime.h>
+
+#include "../../src/general.h"
+
+#define EPS_RESTART_COUNT 10
+#define EPS_RESET_REQUEST 7
+#define EPS_RESET_REASON 11
+
+#define OBC_RESTART_COUNT 8
+#define OBC_RESET_REQUEST 5
+#define OBC_RESET_REASON 9
```

Comment:
You don't need these constants right?

Reply:
nope, overlooked that earlier- removed now